src/main/groovy/com/smartbear/swagger/assertion/ValidStatusCodesAssertionCreator.groovy

```diff
@@ -0,0 +1,16 @@
+package com.smartbear.swagger.assertion
+
+import com.eviware.soapui.impl.wsdl.teststeps.RestTestRequestStep
+import com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion
+import io.swagger.models.Response
+
+import static com.eviware.soapui.security.assertion.ValidHttpStatusCodesAssertion.LABEL
+
+
+class ValidStatusCodesAssertionCreator implements AssertionCreator {
+    @Override
+    void createAssertion(RestTestRequestStep restTestRequestStep, Map<String, Response> responseMap) {
+        ValidHttpStatusCodesAssertion assertion = (ValidHttpStatusCodesAssertion) restTestRequestStep.addAssertion(LABEL)
+        assertion.codes = responseMap.keySet().join(',')
```

Comment:
Also - judging from http://swagger.io/specification/#headersObject - we should be able to add assertions that validate that the specified headers exist in the response!?